### PR TITLE
Properly ignore installPlanStepId errors

### PR DESCRIPTION
**Pull request by aswanson-nr**
# Summary

Some logic to ignore install plan step id errors was lost, this re-adds it.
These errors occur when a new quickstart is created along with its install plan because the database has no idea that the install plan is part of the PR.

### CODE REVIEW

**Review by josephgregoryii — COMMENTED** [review]
Looks great! One question commented below

Also, Is there a reason why we are not checking this at the `validate-install-plan` workflow

File: utils/create_validate_pr_quickstarts.ts [hunk 1]
```diff
@@ -14,20 +14,52 @@ import {
...
+  NerdGraphResponseWithLocalErrors<QuickstartMutationResponse> & {
+    name: string;
+  };
+
+// filter out errors where install plan id does not exist
+const installPlanErrorExists = (error: Error | NerdGraphError): boolean =>
+  'extensions' in error &&
+  error?.extensions?.argumentPath?.includes('installPlanStepIds') &&
+  error?.message?.includes('contains an install plan step that does not exist');
+
```
[the remark this answers is not in the document]
Comment (aswanson-nr) [reply utils/create_validate_pr_quickstarts.ts:32]: Good call! I think we can totally remove that

[the remark this answers is not in the document]
Comment (aswanson-nr) [reply utils/create_validate_pr_quickstarts.ts:32]: I was incorrect, it's needed to type guard for `NerdGraphError`.

**Review by zstix — APPROVED** [review]

File: utils/create_validate_pr_quickstarts.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (josephgregoryii) [reply utils/create_validate_pr_quickstarts.ts:32]: Ahhh, makes sense

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**